### DISCUSSION TITLE
completely uninstall

### DISCUSSION THREAD

**Discussion by bcburnett** [audiamus/BookLibConnect#119] [Q&A]
I recently lost a hard drive that had my download and export folders on it. I tries uninstalling and reinstalling the program but the individual setting remained and the profile button is greyed out and when i push the export... button i get an error.

**Comment by audiamus** [comment]
All user data is stored at `C:\Users\<user>\AppData\Local\audiamus\BookLibConnect`. 

The configuration, which hosts your Audible access token, is encrypted. To protect this sensitive data, decryption only works as long as essential hardware of your machine remains unchanged. It should not be affected by a new hard drive, only by motherboard or CPU change. usersettings.json can be modified with any text editor.

If nothings works, delete everything in `C:\Users\<user>\AppData\Local\audiamus\BookLibConnect` and start afresh.
